packages/ui/src/SliderFormik.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement, useCallback } from 'react'
+import { FieldValidator, useField } from 'formik'
+import { Slider, SliderExtraProps } from './Slider'
+import { getFieldError } from './utils/formik'
+
+export type SliderFormikProps<V extends object> = SliderExtraProps & {
+  name: keyof V
+  validate?: FieldValidator
+}
+
+export const SliderFormik = <V extends object>({ name, validate, ...props }: SliderFormikProps<V>): ReactElement => {
+  const [field, meta, { setValue, setTouched }] = useField<number | [number, number]>({
+    name,
+    validate,
+  })
+
+  const onChange = useCallback(
+    (newValue: number | [number, number]) => {
+      setValue(newValue)
+      setTouched(true)
```

Comment:
`setTouched` runs validation with **previous** value here. From documentation:

> Set touched imperatively. Calling this will trigger validation to run if validateOnBlur is set to true (which it is by default). You can also explicitly prevent/skip validation by passing a second argument as false.

So I suggest following:
```suggestion
      setTouched(true, false)
```

Reply:
I think this https://github.com/hazelcast/frontend-shared/pull/115 should resolve it. WDYT?

Reply:
@aigoncharov Yes, I didn't notice it in other places, nice catch ;)